apps/prairielearn/assets/scripts/esm-bundles/hydrated-components/PublishingForm.ts

```diff
@@ -0,0 +1,5 @@
+import { registerHydratedComponent } from '@prairielearn/preact/hydrated-component';
+
+import { PublishingForm } from '../../../../src/pages/instructorInstanceAdminPublishing/components/PublishingForm.js';
```

Comment:
Nit, can we call this `CourseInstancePublishingForm` since this is living in the shared `hydrated-components/` namespace?